docs/en_US/Release.md

```diff
@@ -1,5 +1,31 @@
 # ChangeLog
 
+## Release 0.9 - 7/1/2019
+
+### Major Features
+* General NAS programming interface
+    * Add `enas-mode` and `oneshot-mode` for NAS interface: [PR #1201](https://github.com/microsoft/nni/pull/1201#issue-291094510)
+* [Gaussian Process Tuner with Matern kernel](./GPTuner.md) 
+
+* Multiphase interface
```

Comment:
For multiphase part update, my draft:

* Multiphase experiment
OpenPAI training serivice now supports multiphase experiment.
Following builtin tuners can be used as multiphase tuner:
TPE, Random Search, Anneal, Naïve Evolution, SMAC,Network Morphism,Metis Tuner
For details, please refer to [Write a tuner that leverages multi-phase](./MultiPhase.md#write-a-tuner-that-leverages-multi-phase)

Reply:
@chicm-ms
fixed in https://github.com/microsoft/nni/pull/1232/commits/65188e735df1c4613c44495383250bc1a46eb3fd

Reply:
@chicm-ms only PAI mode support multiphase? how about local, remote and kubeflow?

Reply:
**Multiphase experiment supports**
- Added new training service support for multiphase experiment: PAI mode supports multiphase experiment since v0.9.
- Added multiphase capability for the following builtin tuners:
TPE, Random Search, Anneal, Naïve Evolution, SMAC, Network Morphism, Metis Tuner.

For details, please refer to Write a tuner that leverages multi-phase

Reply:
updated in https://github.com/microsoft/nni/pull/1232/commits/d6bea8040149ef3945442f677c28d02c8187eb96